src/ui/Roster.ts

```diff
@@ -79,6 +84,78 @@ export default class Roster {
          spanElement.text(msg);
       }
    }
+   
+   private initCollapisible()
+   {
+      let coll = $(".jsxc-collapsible");
```

Comment:
To be consistent with the rest of the code and improve readability, we should rename this to `collapsibleElement`.

Reply:
renamed coll to collapsibleElement